examples/empty_network_listener.cpp

```diff
@@ -114,22 +117,22 @@ class EmptyNetworkListener : public NetworkListener {
 
   TResultOpt removeTrack(TSessionID const &sid,
                          TTrackID const &trkid) override {
-    cout << "Session ID: " << sid << endl;
-    cout << "Track ID: " << trkid << endl;
+    LOG(INFO) << "Session ID: " << sid;
+    LOG(INFO) << "Track ID: " << trkid;
     return nullopt;
   }
   TResultOpt moveTrack(TSessionID const &sid,
                        TTrackID const &trkid,
                        QueueType type) override {
-    cout << "Session ID: " << sid << endl;
-    cout << "Track ID: " << trkid << endl;
+    LOG(INFO) << "Session ID: " << sid;
+    LOG(INFO) << "Track ID: " << trkid;
     return nullopt;
   }
 };
 
 int main(int argc, char *argv[]) {
   if (argc != 2) {
```

Comment:
`initLogging` should be called here too

Reply:
can't call `initLoggingHandler()` here, since `ConfigHandler` is uninitialized at this point (`ConfigHandler::setConfigPath()` has not been called yet)
fixed it by using `cerr` here and calling `initLoggingHandler` later, once ConfigHandler is okay

Reply:
fine. i feel like printing the usage with cerr is better anyway